src/main/java/org/folio/consortia/exception/ResourceNotFoundException.java

```diff
@@ -2,7 +2,7 @@
 
 public class ResourceNotFoundException extends RuntimeException {
 
-  private static final String NOT_FOUND_MSG_TEMPLATE = "Object with %s [%s] was not found";
+  public static final String NOT_FOUND_MSG_TEMPLATE = "Objects with %s [%s] not found";
```

Comment:
Need to correct message in the karate test in this case in this file: https://github.com/folio-org/folio-integration-tests/blob/master/acquisitions/src/main/resources/thunderjet/mod-consortia/features/tenant.feature

Reply:
fixed in karate tests https://github.com/folio-org/folio-integration-tests/pull/1006